plugins/services/src/js/containers/services/ServicesContainer.js

```diff
@@ -522,12 +510,38 @@ class ServicesContainer extends React.Component {
           routes={routes}
           services={filteredServices.getItems()}
           serviceTree={item}>
-          {this.getModals(item)}
           {children}
+          {this.getModals(item)}
         </ServiceTreeView>
       );
     }
 
+    if (item instanceof Pod) {
+      return (
```

Comment:
basically same comment here, extracting the condition block to a method will help a bunch.